app/controllers/planning_applications_controller.rb

```diff
@@ -340,12 +340,4 @@ def ensure_user_is_reviewer
   def ensure_constraint_edits_unlocked
     render plain: "forbidden", status: :forbidden and return unless @planning_application.can_validate?
   end
-
-  def documents_validated_at_missing?
```

Comment:
![image](https://user-images.githubusercontent.com/9452321/144418382-97906631-c07c-43d5-8dc7-3a985788cf92.png)
this was in place to prevent the application crashing when only one data param is entered, if we are removing it then we need a replacement validation.

Reply:
![image](https://user-images.githubusercontent.com/9452321/144418505-2b8e89b2-b199-419a-a61a-ab72a7d63670.png)

Reply:
op nevermind it was the previous method that we already removed that validated this step - just realised this is also broken in staging